packages/@react-types/button/src/index.d.ts

```diff
@@ -95,7 +95,7 @@ interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
 }
 
 export interface AriaButtonProps<T extends ElementType = 'button'> extends ButtonProps, LinkButtonProps<T>, AriaBaseButtonProps {}
-export interface AriaToggleButtonProps<T extends ElementType = 'button'> extends ToggleButtonProps, Omit<AriaBaseButtonProps, 'aria-current'>, AriaButtonElementTypeProps<T> {}
+export interface AriaToggleButtonProps<T extends ElementType = 'button'> extends ToggleButtonProps, Omit<AriaBaseButtonProps, 'aria-current' | 'form' | 'formAction' | 'formEncType' | 'formMethod' | 'formNoValidate' | 'formTarget' | 'name' | 'value'>, AriaButtonElementTypeProps<T> {}
```

Comment:
i know we wanted to get rid of the "submit" but that's under the `type` prop which we've had released for awhile so not sure if we wanted to get rid of it in case that might be breaking?

Reply:
feels like we maybe able to classify it as a bug fix but happy for others to weight in